### ISSUE TITLE
35896463: Manually setting accessibilityElements makes it difficult to drag to focus large UILabels with VoiceOver

### ISSUE THREAD

**Issue by openradar-mirror** [lionheart/openradar-mirror#19031]
#### Description

Create a stack of accessible elements (say a table view) and place a large UILabel (font size 200 or so) above them. Turn on VoiceOver and attempt to tap or drag to select the large label. It should be fairly easy. Next set the accessibilityElements of the view manually (either with the large label coming first in the array or last) and observe that it's very hard/impossible to select by dragging or tapping. Shrinking the label's accessibility Frame, or using a smaller font seems to be less problematic.

To use sample project, run with VO running. Attempt to tap 'X' targets, or drag to select. Note that it's very easy. Tap button to set them manually. Attempt again. Observe that it's possible to tap the smaller one to highlight it but dragging to select either is difficult.

Sample project:
https://github.com/warpling/LargeLabelAccessibilityTest

-
Product Version: 
Created: 2017-12-07T00:49:16.742790
Originated: 2017-12-06T00:00:00
Open Radar Link: http://www.openradar.me/35896463
